### DISCUSSION TITLE
Show PR conflicts

### DISCUSSION THREAD

**Discussion by henrikbrixandersen** [community/community#14894] [Mobile]
The web GUI shows when a PR has conflicts. It would be nice if the mobile app could do the same.

**Comment by stevepopovich** [comment]
Hi Henrik! Thanks for the feedback! I filed the request and passed it on to the team for prioritization.

**Reply by henrikbrixandersen** [reply]
Any updates on this? ☺️

**Reply by stevepopovich** [reply]
No sorry, still in our backlog for consideration.
